ros_pospac_bridge:
  ros__parameters:
    gps_data_file: "/home/javadibrahimli/ros2_ws/route2.txt"
    mgrs_origin: "35TPF6645943620"  # Replace with your MGRS origin
    origin_altitude: 42.743

    # Publisher Control (True/False Flags)
    start_gps_publisher: true
    start_imu_publisher: true
    start_pose_publisher: true
    start_pose_array_publisher: true
    start_twist_publisher: true
    start_pose_stamped_publisher: true

    # Transformation Settings
    lidar_to_gnss_transform:
      x: 0.0
      y: 0.0
      z: -0.08
      roll: 0.007237
      pitch: -0.0072323
      yaw: 3.1334945

    lidar_to_base_link_transform:
      x: 0.96
      y: -0.01
      z: 1.1
      roll: 0.0
      pitch: 0.0
      yaw: -3.0965926
